tests/test_c8ql.py

```diff
@@ -10,7 +12,54 @@
     C8QLQueryKillError,
     C8QLQueryValidateError,
 )
-from tests.helpers import assert_raises, extract
+from tests.helpers import assert_raises, extract, generate_col_name
+
+
+def test_export_data_query(client, docs):
+    collection_name = generate_col_name()
```

Comment:
I think we should try to use tst_fabric as much as we can by adding tst_fabric_name in params and then inside the test add "client._tenant.useFabric(tst_fabric_name)".